core/pta/device.c

```diff
@@ -72,7 +78,9 @@ static TEE_Result invoke_command(void *pSessionContext __unused,
 {
 	switch (nCommandID) {
 	case PTA_CMD_GET_DEVICES:
-		return get_devices(nParamTypes, pParams);
+		return get_devices(nParamTypes, pParams, PTA_CMD_GET_DEVICES);
+	case PTA_CMD_GET_DEVICES_SUPP:
+		return get_devices(nParamTypes, pParams, PTA_CMD_GET_DEVICES_SUPP);
```

Comment:
You have changed here but not `get_devices()` nor `add_ta()`. I doubt it will work well :wink: